### ISSUE TITLE
Cannot assign issue to user None anymore

### ISSUE THREAD

**Issue by WayneMDB** [pycontribs/jira#907]
When trying to assign an issue to None, the _get_user_accountid() function generates an error trying to search for user None. Instead, that function should start with:

```
    if user is None:
        return None # Works fine, though perhaps returning a string of "None" is cleaner?
```

To reproduce, just call `jira.assign_issue(jira_issue, None)`

This is in the latest git master.

**Comment by Addono** [comment]
Probably introduced in a4e6b1b202f3083ebd923b1ff6a7fc43455e687f

**Comment by adehad** [comment]
Fixed in #1088
